Packages/Ink/Editor/Core/Ink Settings/InkSettings.cs

```diff
@@ -68,17 +68,6 @@ public string templateFilePath {
 				else return AssetDatabase.GetAssetPath(templateFile);
```

Comment:
the `templateFile` should probably change type to the ScriptableObject imported by the InkImporter